src/translate.js

```diff
@@ -65,6 +65,10 @@ export default function translate(namespaces, options = {}) {
             };
 
             this.i18n.on('initialized', initialized);
+
+            // In case of race condition, that 'initialized' never comes - do immediately 
+            // check ready state + if i18n is initialized.
+            setTimeout(() => !this.state.ready && this.i18n.isInitialized && ready());
```

Comment:
As you also got i18next.options not set -> it really seems you do not call i18next.init before rendering to dom.